.github/workflows/linter.yml

```
@@ -0,0 +1,15 @@
+name: Linter
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Lint
+      run: |
+        npm install
+        npm run lint
```

We need to build, too, otherwise we won't see errors.

```suggestion
        npm run lint
        gatsby build
```